packages/cli/src/tools/config/readConfigFromDisk.js

```diff
@@ -123,11 +125,5 @@ export function readLegacyDependencyConfigFromDisk(
     )} is using deprecated "rnpm" config that will stop working from next release. Consider upgrading to the new config format.`,
```

Comment:
while we're at it, this message is plain wrong, can we change it?
```suggestion
     )} is using deprecated "rnpm" config that will stop working from next release. Please notify its maintainers about it.`,
```

Reply:
this suggestion caused a lint error, please run `yarn lint --fix`